api4/cloud.go

```diff
@@ -125,38 +125,38 @@ func changeSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	bodyBytes, err := ioutil.ReadAll(r.Body)
 	if err != nil {
-		c.Err = model.NewAppError("Api4.changeSubscription", "api.cloud.app_error", nil, err.Error(), http.StatusBadRequest)
+		c.Err = model.NewAppError("Api4.changeSubscription", "api.cloud.app_error", nil, "", http.StatusBadRequest).Wrap(err)
```

Comment:
Just a thought. Since we are moving to the wrapping approach, we might want to create a custom "go vet" check that flags any new code added with the `err.Error()` field passed.

And then with the next breaking change, we can change the function signature to pass the error itself.

Reply:
Love the idea!